SecurityPkg/Include/Protocol/MuTcg2Protocol.h

```diff
@@ -38,11 +38,35 @@ EFI_STATUS
   IN TPML_DIGEST_VALUES   *DigestList,
   IN EFI_TCG2_EVENT       *Event
   );
+
+/**
+  The EFI_MU_TCG2_PROTOCOL MuEventInit function call provides callers with
+  an interface for initializing events before they are logged.

```

Comment:
I suggest you drop the symbol name details. The description itself will usually be associated with the code (e.g. in Doxygen and VS Code) and it's easier to read just the description.

```suggestion
  Initializes a TCG PCR Event structure so it is populated for further use such as logging.
```

Reply:
I know this is based on the example above. You don't need to modify it.

Reply:
I also updated my suggestion to include a clarification that this only modifies the `Event` structure provided. I think that's helpful because the actual operation the function performs feels like something that would be in more of a helper library versus a protocol.